proposals/simd/SIMD.md

```diff
@@ -6,6 +6,20 @@ current popular instruction set architectures.
 
 See also [The binary encoding of SIMD instructions](BinarySIMD.md).
 
+## Motivation
+
+WebAssembly aims to take advantage of [common hardware capabilities](https://github.com/WebAssembly/design/blob/master/Portability.md#assumptions-for-efficient-execution)
+for near native speed. The motivation for this proposal is to introduce
+WebAssembly operations that map to commonly available SIMD instructions
+in hardware. 
+
+SIMD instructions in hardware work by performing simultaneous computations over
+packed data in one instruction. These are commonly used to improve performance
+for multimedia applications. The SIMD proposal in WebAssembly is comprised of
+a portable subset of operations that in most cases map to commonly used hardware
```

Comment:
Reword this a bit? It says "portable subset", which implies of the various hardware instructions, but it might be clearer to say it explicitly.

Reply:
Reworded now, WDYT?

Reply:
lgtm!